storage_service/locations/api/search/router.py

```diff
@@ -0,0 +1,159 @@
+import django_filters
+from rest_framework import routers, serializers, viewsets, filters
+from rest_framework.decorators import list_route
+from rest_framework.response import Response
+
+from django.db.models import Sum
+
+from locations import models
+
+
+class CaseInsensitiveBooleanFilter(django_filters.Filter):
+    """
+    This allows users to query booleans without having to use "True" and "False"
+    """
+    def filter(self, qs, value):
+        if value is not None:
+            lc_value = value.lower()
+            if lc_value == "true":
+                value = True
+            elif lc_value == "false":
+                value = False
+            return qs.filter(**{self.name: value})
```

Comment:
Just tested, looks like it's ignored if a supported value isn't passed. Maybe we want this it to raise an error instead?